### ISSUE TITLE
More precise/accurate GMS version estimation

### ISSUE THREAD

**Issue by Grossley** [UnderminersTeam/UndertaleModTool#1187]
### Which component should be improved?

UI/UX

### Describe your feature suggestion in more detail

I think it would be possible to get more accurate information by comparing the game compile timestamp to known GMS1/2 releases as people usually update immediately, then cross referencing that to the version flags to check for validity. If the check succeeds then we can use the more precise version, otherwise, we can default to the initial estimate.

**Comment by Miepee** [comment]
I don't think this would actually help, especially now that multiple branches (LTS and non-LTS) exist.
Anyone objecting to closing this issue?

**Comment by colinator27** [comment]
Yeah, this is probably not something we want.
